acceptance/localcluster/localcluster.go

```diff
@@ -564,9 +564,11 @@ func (l *Cluster) stop() {
 				log.Infof("node %d: stderr at %s", i, file)
 			}
 		} else if crashed {
+			// Don't remove crashed containers - could be useful for post-mortem.
```

Comment:
We do remove the volumes when the container is removed: https://github.com/cockroachdb/cockroach/blob/master/acceptance/localcluster/docker.go#L167

And it isn't disk space that I'm concerned about, just general cleanliness.